test/test-index.js

```diff
@@ -197,3 +220,73 @@ test.cb('The task receives the cumulus_config property', (t) => {
   return cumulusMessageAdapter
     .runCumulusTask(businessLogic, inputEvent, context, t.end);
 });
+
+test('GetMessageGranules returns empty array if no granules are found', (t) => {
+  const messageGranules = getMessageGranules(testContext.inputEvent);
+
+  t.deepEqual(messageGranules, []);
+});
+
+test('GetMessageGranules returns granules if they are in the payload', (t) => {
+  const messageGranules = getMessageGranules(testContext.granuleInputJson);
+
+  t.deepEqual(messageGranules, [
+    'MOD09GQ.A2016358.h13v04.006.2016360104606',
+    'MOD09GQ.A2016358.h13v04.007.2017'
+  ]);
+});
+
+test('GetMessageGranules returns granules if they are in the meta.input_granules', (t) => {
+  const messageGranules = getMessageGranules(testContext.inputGranuleInputJson);
+
+  t.deepEqual(messageGranules, [
+    'MOD09GQ.A2016358.h13v04.006.2016360104606',
+    'MOD09GQ.A2016358.h13v04.007.2017'
+  ]);
+});
+
+test('GetMessageGranules returns CMA granules if they are in the CMA event payload', (t) => {
+  const messageGranules = getMessageGranules(testContext.paramGranuleInputJson);
+
+  t.deepEqual(messageGranules, [
+    'MOD09GQ.A2016358.h13v04.006.2016360104606',
+    'MOD09GQ.A2016358.h13v04.007.2017'
+  ]);
+});
+
+test('GetMessageGranules returns granules if they are in the CMA event meta.input_granules',
```

Comment:
ditto the above.

Reply:
Seems like not based on Lauren's comment above